packages/gatsby-recipes/src/providers/gatsby/__snapshots__/plugin.test.js.snap

```diff
@@ -3291,7 +3291,7 @@ exports[`gatsby-plugin resource e2e plugin resource test with hello world starte
 exports[`gatsby-plugin resource e2e plugin resource test with hello world starter: GatsbyPlugin update 1`] = `
 Object {
   "_message": "Installed gatsby-source-filesystem in gatsby-config.js",
-  "description": "Gatsby plugin which parses files within a directory for further parsing by other plugins",
+  "description": null,
```

Comment:
I investigated this more & this is correct. We don't installed the packages for these & it never should have worked. The only reason we got descriptions before was due to a bug in npm/package.js where it wasn't using the `root` value https://github.com/gatsbyjs/gatsby/pull/26699/files#r479681061